tests/boot.pm

```diff
@@ -3,6 +3,7 @@
 
 use base 'basetest';
 use strict;
+use warnings;
```

Comment:
We already established that we don't want to have the explicit `use strict` and `use warnings` in test modules. I guess you can find a related discussion already in other pull requests in this repo. Please see http://open.qa/docs/#_test_module_interface for the suggested style. So basically replace the above three lines with

```
use Mojo::Base 'basetest';
```

Reply:
i know but i found it as that and i didnt want to interfere to the approach. i will change them